### DISCUSSION TITLE
Can I install rust on Windows VM ?

### DISCUSSION THREAD

**Discussion by ekokurniadi** [codemagic-ci-cd/codemagic-docs#1837] [Q&A] [answered]
Hi, i have a flutter desktop application with some feature with rust. I want to use codemagic for build msix, but i got error cause rust not installed on windows vm. Can i install rust on windows vm ?

**Comment by mikhail-tokarev** [comment, marked as answer]
Hi
sure, you can use [Chocolately](https://community.chocolatey.org) to install software on Windows, for instance `choco install rust`

**Reply by ekokurniadi** [reply]
thank you 👍
